manticore/core/cpu/arm.py

```diff
@@ -1258,3 +1258,15 @@ def DMB(cpu, *operands):
     @instruction
     def LDCL(cpu, *operands):
         """Occasionally used in glibc (longjmp in ld.so). Nop under our execution model."""
+
+    @instruction
+    def UQSUB8(cpu, dest, op1, op2):
+        src1 = op1.read()
+        src2 = op2.read()
+        result = 0
+        for i in (range(op1.size / 8)):
```

Comment:
We should do it as `op1.size // 8` so that we are compatible with both PY2 and Py3.
We could also add `from __future__ import division` at the top.

Reply:
Ok, here you go